pkg/services/authz/client.go

```diff
@@ -105,7 +106,7 @@ func newInProcLegacyClient(server *legacyServer) (authzlib.MultiTenantClient, er
 		&authzlib.MultiTenantClientConfig{},
 		authzlib.WithGrpcConnectionLCOption(channel),
 		// nolint:staticcheck
-		authzlib.WithNamespaceFormatterLCOption(authnlib.OnPremNamespaceFormatter),
+		authzlib.WithNamespaceFormatterLCOption(claims.OrgNamespaceFormatter),
```

Comment:
These changes are due to authlib's changes, unrelated to this PR, but needed for building